### PR TITLE
:construction_worker: add github ci

### PR DESCRIPTION
**Pull request by heavenboy8**

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8"]
```
Comment (ThibsG) [inline .github/workflows/cd.yml:14]: why not 3.9 also?

File: .github/workflows/backend.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        - /var/run/docker.sock:/var/run/docker.sock
+        - /tmp:/tmp
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
```
Comment (ThibsG) [inline .github/workflows/backend.yml:26]: no matrix used in this GA file

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        - /tmp:/tmp
+    needs: check
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
```
Comment (ThibsG) [inline .github/workflows/ci.yml:82]: no matrix set on this job

File: .github/workflows/scheduled.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: main
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
```
Comment (ThibsG) [inline .github/workflows/scheduled.yml:24]: no matrix

**Review by ThibsG — COMMENTED** [review]
Thanks !

File: .github/workflows/cd.yml (hunk 1 above)
Comment (ThibsG) [reply .github/workflows/cd.yml]: Ah ok! Got it, it has nothing to do with any targeted version on Pypi right?

Comment (heavenboy8) [reply .github/workflows/cd.yml]: nothing

**Review by ThibsG — APPROVED** [review]

**Pull request merged**